### PR TITLE
Better matrix setup for tox

### PR DESCRIPTION
**Pull request by kumar303**
I learned a much better way to set up multiple versions of dependencies in tox.

### CODE REVIEW

**Comment by kumar303** [comment]
@frankoid hmm, it looks like https://travis-ci.org/brighthouseinteractive/django-encrypted-cookie-session isn't set up. Or maybe I have the link wrong and pull requests just aren't configured?

**Comment by frankoid** [comment]
Travis wasn't enabled at all for this project. I've just enabled it.

**Comment by frankoid** [comment]
Thanks for this PR :)

I think you might need to specify `basepython` in tox.ini as per https://tox.readthedocs.org/en/latest/config.html#generating-environments-conditional-settings.

I'd prefer it if tox.ini used `django18` instead of `django1.8` (etc) for consistency with the tox docs and also our (Bright Interactive's) other packages.

File: tox.ini [hunk 1]
```diff
@@ -1,74 +1,24 @@
...
-[testenv:py26-django16]
-basepython=python2.6
-deps={[django16]deps}
+    django1.4: Django>=1.4,<1.5
+    django1.5: Django>=1.5,<1.6
+    django1.6: Django>=1.6,<1.7
+    django1.7: Django>=1.7,<1.8
+    django1.8: Django>=1.8,<1.9
+setenv =
+    PYTHONDONTWRITEBYTECODE=1
```
Comment (frankoid) [inline tox.ini]: what is this for?

Comment (kumar303) [inline tox.ini]: this prevents writing pyc files which I guess isn't a big deal, I forget why I thought that would be a good idea. Generally they make things annoying during development.

**Comment by kumar303** [comment]
ok, `tox.ini` should be fixed up now

**Comment by kumar303** [comment]
btw, the `basepython` resolves by default to what we want but I added it in anyway.

**Pull request merged**